Packs/PANOSPolicyOptimizer/pack_metadata.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "PAN-OS Policy Optimizer",
+    "name": "PAN-OS Policy Optimizer (beta)",
     "description": "This integration introduces Policy Optimizer and DAG features that are not available through the regular PAN API",
     "support": "xsoar",
```

Comment:
Need to update the support for community as well

Reply:
Also please fix validation - add new release notes and bump the pack version

Reply:
fixed - updated the cuuerntVersion and support in metadata file.